_posts/2021-06-22-choosing-right-transformation.md

```diff
@@ -0,0 +1,114 @@
+
+Using the word properly, a ___projection___ spreads this 3D model of the surface of the earth onto a flat, two-dimensional plane. These are the projections we all know and love (or [hate](https://xkcd.com/977))—Robinson, Mercator, UTM, etc.
+
+## Irreconcilable Differences
+
+The WGS84 datum and the NAD83 datum started out identical. However, because WGS84 and NAD83 are tied and adjusted to different sets of points on the earth's surface (NAD83 focuses on the North American area, while WGS84 focuses on the entire world), they have drifted apart over time like a celebrity marriage.
+
+Because a projection using NAD83 and another using WGS84 are no longer starting from the same spot, any projected coordinates calculated in both no longer match. While the difference is slight on a planetary scale, it often works out to around a meter here in Utah.
+
+## Maybe Not So Irreconcilable
+
+Fortunately, we can use [___transformations___](https://desktop.arcgis.com/en/arcmap/10.3/guide-books/map-projections/choosing-an-appropriate-transformation.htm) to figure out the differences between datums. Each different transformation uses its own methods and starting reference points to translate coordinates.
+
+The trick is choosing the right transformation. The pitfall is that the default is usually wrong if you're going from NAD83 to WGS84.
+
+___Our recommended transformation between NAD83 and WGS84 in Utah___ is `NAD_1983_To_WGS_1984_5`. Other transformations may work better for other geographic areas or for other projections using different datums.
+
+![NAD 1983 to WGS 1984 5 is the right transformation]({% link images/transformations_proper.jpg %}){: .flex .flex--center}
+
+## UTM, SGID, and You
+
+Using the right transformation with SGID data has become more important with the proliferation of AGOL-hosted feature services and the SGID layers shared via [opendata.gis.utah.gov](https://opendata.gis.utah.gov).
+
+Internally, all the SGID layers hosted by UGRC are kept in the UTM 12N NAD83 projection (EPSG 26912). As its name suggests, this uses the NAD83 datum. If you've downloaded data from us in the past or used our old direct SDE connection, you got UTM data. Our new [Open SGID](https://gis.utah.gov/sgid/#open-sgid) database offering also serves out data in UTM.
+
+We've made a large push over the last year or so to host all our SGID data in ArcGIS Online as well. This allows you to search for data right in ArcGIS Pro or in AGOL webmaps. It also allowed us to set up [opendata.gis.utah.gov](https://opendata.gis.utah.gov), which in turn drives all the download links on our [data pages](https://gis.utah.gov/data/#data-categories).
```

Comment:
**chore**: internal links should use the [link syntax](https://github.com/agrc/gis.utah.gov/#link-to-post)